sdks/python/apache_beam/runners/portability/construction_service.py

```diff
@@ -0,0 +1,116 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
```

Comment:
IIUC This is same as above but implemented for Python. Is this correct ?

Reply:
Exactly. Makes for easier, self-contained testing (and we'll want to be able to go both ways).